.github/workflows/tests.yml

```diff
@@ -21,29 +21,38 @@ jobs:
           - os: ubuntu-20.04
```

Comment:
I think it is time to split this into multiple tests, I'll do it in a separate PR

Reply:
Thought the same. If you do it I review.